### DISCUSSION TITLE
Simultaneous use of external wifi module for broadcast, and built in wifi chip for data?

### DISCUSSION THREAD

**Discussion by gworkman** [svpcom/wfb-ng#91] [Questions]
Hi,

Great work with this project! Works well. I was wondering if for the Raspberry Pi 3 (or other devices) it is possible to use Wifibroadcast to transmit live video over the external wifi module while at the same time still using the built-in wifi chipset to connect to the internet?

Thanks!

**Comment by svpcom** [comment]
Yes, it is possible

**Comment by svpcom** [comment]
Just add wifi card used by WFB to `unmanaged-devices` in NetworkManager.conf

**Comment by ghost** [comment]
@gworkman Doing wifibroadcast on Navio perhaps?
